drift/lib/src/runtime/query_builder/schema/table_info.dart

```diff
@@ -28,6 +28,16 @@ mixin TableInfo<TableDsl extends Table, D> on Table
   @override
   Set<Column> get primaryKey => $primaryKey;
 
+  /// The unique key of this table. Can be empty if no custom primary key has
+  /// been specified.
+  ///
+  /// Additional to the [Table.primaryKey] columns declared by an user, this
+  /// also contains auto-increment integers, which are primary key by default.
+  Set<GeneratedColumn> get $uniqueKey => const {};
+
+  @override
+  Set<Column> get uniqueKey => $uniqueKey;
```

Comment:
I think the `$primaryKey` indirection is a left-over from early days and doesn't need to be replicated here :D

I think we can just do `@override Set<GeneratedColumn> get uniqueKey;` here to make the return type more specific, but it looks like there doesn't have to be a `$uniqueKey` getter.